tests/integration/golang/fixtures/run.go

```diff
@@ -78,12 +94,44 @@ func (f RunFixtures) CreateRuns(
 		if err != nil {
 			return nil, err
 		}
+		tag := models.Tag{
+			Key:   "my tag key",
+			Value: "my tag value",
+			RunID: run.ID,
+		}
+		err = f.CreateTag(ctx, tag)
+		if err != nil {
+			return nil, err
+		}
+		run.Tags = []models.Tag{tag}
+
+		err = f.CreateMetrics(ctx, run, 2)
+		if err != nil {
+			return nil, err
+		}
+
 		runs = append(runs, run)
 	}
 	return runs, nil
 }
 
-// GetTestRuns fetches all runs for an experiment
+// GetTestRun fetches one run.
+func (f RunFixtures) GetTestRun(
```

Comment:
I totally confused. We have `CreateExampleRun` and `CreateRun` can we please move forward with only on function?

Reply:
`CreateRun` persists a Run provided as parameter, but `CreateExampleRun` does not require a Run parameter, it creates for you.